build/__javaVersions.sh

```diff
@@ -1,4 +1,5 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
 JAVA_VERSION='17.0.2'
+JAVA_VERSION11='11.0.14'
```

Comment:
Can we have consistent variable names: JAVA11_VERSION and JAVA17_VERSION for them

Reply:
I believe this is coming from @william-msft's PR (https://github.com/microsoft/Oryx/pull/1348/files), and that it's onlky a temporary change.
That said, I imagine this will create a conflict when merging. Should we remove it from this PR?

Reply:
I am using the constant `JAVA_VERSION11` in my dockerfile. Hence, I do need it in my PR :| 
I could change my var name to `JAVA11_VERSION` if that helps.

Reply:
@samruddhikhandale Thanks. Then it's likely easier to keep it as it is, and once @william-msft merges his PR to remove it from yours while merging changes.

Reply:
Sure. Sounds good. thanks 👍